internal/authz/gitlab/sudo.go

```diff
@@ -212,6 +212,12 @@ func (p *SudoProvider) FetchUserPerms(ctx context.Context, account *extsvc.Accou
 	return listProjects(ctx, client)
 }
 
+// FetchUserPermsByToken is the same as FetchUserPerms but only requires a token.
```

Comment:
Making this same as the doc string above for `gitlab/oauth.go`:

```suggestion
// FetchUserPermsByToken is the same as FetchUserPerms but it only requires a
// token.
```